staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -409,6 +409,13 @@ type KubeletConfiguration struct {
 	// Default: "10s"
 	// +optional
 	CPUManagerReconcilePeriod metav1.Duration `json:"cpuManagerReconcilePeriod,omitempty"`
+	// TopologyManagerPolicy is the name of the policy to use.
+	// Requires the TopologyManager feature gate to be enabled.
```

Comment:
"Policies other than `none` require the TopologyManager feature gate to be enabled."